operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_CPS.md

```diff
@@ -1,37 +1,35 @@
 # Create a Session Template to Boot Compute Nodes with CPS
 
 When [compute nodes](../../glossary.md#compute-node-cn) are booted, the [Content Projection Service (CPS)](../../glossary.md#content-projection-service-cps) and
-[Data Virtualization Service (DVS)](../../glossary.md#data-virtualization-service-dvs) project the root file system \(`rootfs`\) over the network to the compute nodes by default.
+[Data Virtualization Service (DVS)](../../glossary.md#data-virtualization-service-dvs) project the root file system (`rootfs`) over the network to the compute nodes by default.
 
 Another option when compute nodes are booted is to download their `rootfs` into RAM.
 
 This page covers the appropriate contents for a BOS session template in order to use CPS and DVS.
 
-- [Boot set S3 parameters](#boot-set-s3-parameters)
-- [Boot set `rootfs_provider` parameters](#boot-set-rootfs_provider-parameters)
+- [Boot set `rootfs_provider` parameter](#boot-set-rootfs_provider-parameter)
+- [Boot set `rootfs_provider_passthrough` parameters](#boot-set-rootfs_provider_passthrough-parameters)
   - [`<transport>`](#transport)
   - [`<api_gateway>`](#api_gateway)
   - [`<timeout>`](#timeout)
-  - [`<etag>`](#etag)
   - [`<interface>[,<interface>][,<interface>]...`](#interfaceinterfaceinterface)
   - [`<ramroot>`](#ramroot)
   - [Example `rootfs_provider_passthrough`](#example-rootfs_provider_passthrough)
-- [`root=` kernel parameter](#root-kernel-parameter)
+- [Example kernel parameter](#example-kernel-parameter)
+- [Boot set S3 parameters](#boot-set-s3-parameters)
 - [Example session template input file](#example-session-template-input-file)
 - [Creating a BOS session using the new template](#creating-a-bos-session-using-the-new-template)
+- [`root=` kernel parameter](#appendix-root-kernel-parameter)
```

Comment:
With the heading, this link name no longer matches the heading. You might want to fix that.

Reply:
Fixed.